### PR TITLE
Hello 2x

### PR DESCRIPTION
**Pull request by joesimp94**

### CODE REVIEW

**Review by mz106 — CHANGES_REQUESTED** [review]
Please see code comments

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # beansontoast
 
 Hello Joe
+Hello Joe Again
```
Comment (mz106) [inline README.md:4]: Please amend to "Hello Michael"

**Review by mz106 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (joesimp94) [reply README.md]: Okay, changes made

**Review by mz106 — APPROVED** [review]
Good to merge

**Pull request merged**